src/frequently-requested-changes.md

```diff
@@ -0,0 +1,165 @@
+`self.othermethod()`, and accessing a field requires `self.field`. In some
+other languages, such accesses can omit the equivalent of `self.`, and just
+write `othermethod` or `field`, implicitly referencing the "current object".
+People writing code with such method calls or field accesses sometimes ask for
+such shorthand in Rust.
+
+Rust prefers the explicitness of writing `self.` for method calls and field
+accesses, and in most ways treating `self` as a normal object of the type. This
+avoids unexpected calls to the wrong method, makes it easier to distinguish
+methods and free functions, and makes code easier to read. We don't want to
+provide a shorthand that makes this syntax briefer at the expense of this
+clarity and unambiguity.
+
+## Arbitrary custom operator syntax
+
+Rust allows overloading existing operators; for instance, you can implement the
+`Add` trait to overload the `+` operator. However, Rust does not allow creating
+*new* operators.
+
+Some would argue this can make code more readable, if you know what the
+operators mean. If you don't, it makes the code inscrutable.
+
+In general, such a change would substantially raise barriers to entry for Rust
+developers, making Rust code less approachable and less universally
+understandable. We're unlikely to add support for this.
+
+Note that Rust's existing operator overloading is semantic (`impl Add`), not
+syntactic (`impl Plus`), which tends to encourage using overloaded operators
+for the same semantic purposes, rather than for building arbitrary domain-specific languages.
+
+Rust developers seeking to build arbitrary domain-specific languages (DSLs)
+should consider the macro system.
+
+## Numeric overflow checking should be on by default even in release mode
+
+Whenever possible, Rust tries to do the safe thing by default.
+
+Numeric overflow checking (e.g. `1000u16 * 1000u16`) is one case where Rust
+compromised on this: on many targets, numeric overflow checking has high enough
+overhead to hurt performance too much for a wide variety of code. As a result,
+Rust defaults to having overflow checking only for debug builds, while release
+builds have overflow checking off by default. (In release builds, numeric
+overflow wraps, but code cannot count on this behavior, and projects can turn
+on overflow checking in release mode.)
+
+We've thought about this choice many times, and we're open to considering
+changes to this default based on benchmarks. If, on some Rust targets, overflow
+checking adds fairly little overhead on the vast majority of crates, we'd
+consider enabling it by default for those targets.
+
+It would also help to have ways to detect excessive overhead caused by overflow
+checking (e.g. detecting numeric-heavy code) and suggesting the use of
+explicitly non-overflowing numeric types such as `Wrapping`.
+
+## Cross-function type inference
+
+Rust's type inference stops at function boundaries; Rust requires specifying
```

Comment:
well except for `impl Trait` ... are you deliberately not pointing people at that exception?

Reply:
@pnkfelix Happy to add hedging like "generally" if that helps. Wasn't being deliberate so much as trying to cover the common case.